docs/semgrep-assistant/getting-started.md

```diff
@@ -146,12 +154,29 @@ While Assistant Memories is in **public beta**, memories are scoped to remediati
 2. In the **Assistant** section, click the <i class="fa-solid fa-gear"></i> **icon** next to **Customize with memories**.
 3. Click the <i class="fa-solid fa-trash"></i> **icon** to remove the memory.
 
-### Enable priority inbox
+### Use your own OpenAI API key
 
-If [priority inbox](/semgrep-assistant/overview/#priority-inbox), which allows organization admins to receive information on top backlog tasks according to Assistant, isn't enabled for your deployment, you can do so as follows:
+If you want complete control over how OpenAI handles your data, you can use your OpenAI API key instead of Semgrep's. To provide your OpenAI API key:
 
-1. Sign in to Semgrep AppSec Platform, and navigate to **Settings > Deployment**.
-2. In the **Assistant** section, click the **Weekly priority emails** <i class="fa-solid fa-toggle-large-on"></i> if it is not yet enabled.
+1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login?return_path=/manage/projects) and navigate to [<i class="fa-solid fa-gear"></i> **Settings > Deployment**](https://semgrep.dev/orgs/-/settings).
+2. In the **Assistant** section, click the <i class="fa-solid fa-gear"></i> **icon** next to **AI provider**.
```

Comment:
Note that the "**click the** icon..." here is ok because it's not the same as the name of a UI element.

https://developers.google.com/style/word-list#click